internal/command/flags.go

```diff
@@ -31,26 +30,19 @@ var (
 )
 
 func addFlagAPIRoot(fs *flag.FlagSet) {
-	fs.StringVar(&flagAPIRoot, "api-root", "", "location of googleapis repository")
+	fs.StringVar(&flagAPIRoot, "api-root", "", "(Required) location of googleapis repository")
```

Comment:
I don't _think_ this is required (if flagAPIRoot is empty, we clone googleapis)

https://github.com/googleapis/generator/pull/25/files#diff-69d3575d337e6861d23e877e04bbdb99f437a82661f706f055a50e82b1e8282aR81

I think this needs to be fixed:

https://github.com/googleapis/generator/pull/25/files#diff-69d3575d337e6861d23e877e04bbdb99f437a82661f706f055a50e82b1e8282aR54

Reply:
It does behave this way for generate -- I was playing around and found it required for the 'configure' command, but then it's also not yet implemented.

I've updated to reflect the generate command's behavior which is the current focus.